sublime-package.json

```diff
@@ -995,12 +497,24 @@
               "title": "Allow implicit Optional when default parameter value is None",
               "default": true
             },
+            "enableExperimentalFeatures": {
+              "$id": "#/properties/enableExperimentalFeatures",
+              "type": "boolean",
+              "title": "Enable the use of experimental features that are not part of the Python typing spec",
+              "default": true
+            },
             "enableTypeIgnoreComments": {
               "$id": "#/properties/enableTypeIgnoreComments",
               "type": "boolean",
               "title": "Allow \"# type: ignore\" comments",
               "default": true
             },
+            "deprecateTypingAliases": {
+              "$id": "#/properties/deprecateTypingAliases",
+              "type": "boolean",
+              "title": "Treat typing-specific aliases to standard types as deprecated",
+              "default": true
```

Comment:
I've noticed that some `default` values don't actually match default pyright values and reported it as https://github.com/microsoft/pyright/issues/6711